src/test/java/no/unit/nva/model/DtoTest.java

```diff
@@ -0,0 +1,26 @@
+package no.unit.nva.model;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+// Not a final class in order to show where these values methods are used through inheritance.
+@SuppressWarnings("NonFinalUtilityClass")
+public class DtoTest {
+
+    protected static final String JSON_TYPE_ATTRIBUTE = "type";
+    protected static final String USER_TYPE_LITERAL = "User";
+    protected static final String ROLE_TYPE_LITERAL = "Role";
```

Comment:
USER_TYPE_LITERAL and ROLE_TYPE_LITERAL should be moved to the classes where they are used.